hack/generator/pkg/astmodel/gomega_extensions_test.go

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astmodel
+
+import (
+	"fmt"
+
+	"github.com/onsi/gomega/format"
+	"github.com/onsi/gomega/types"
+)
+
+func MatchErrorString(value string) types.GomegaMatcher {
```

Comment:
Why doesn't this work?
https://onsi.github.io/gomega/#matcherrorexpected-interface

Reply:
It seems to support exactly this case already? Or am I missing something?

Reply:
what! I looked for this and somehow didn't find it. deleting my version ...